### DISCUSSION TITLE
(Resolved) Which version of Godot is v0.11.3 really using?

### DISCUSSION THREAD

**Discussion by theofficialgman** [Orama-Interactive/Pixelorama#931] [General]
Release notes say 3.5.2 but I imagine that is incorrect https://github.com/Orama-Interactive/Pixelorama/releases/tag/v0.11.3

**Comment by OverloadedOrama** [comment]
v0.11.3 is using Godot 3.5.2. v1.0 is using Godot 4.1. v0.11.3 is just a bug-fixing maintenance release.

**Reply by theofficialgman** [reply]
is the tag incorrect then or is that branch actually compatible with godot 4 and godot 3? The history of that tag indicates your godot 4 PR is already included and was not reverted

**Reply by OverloadedOrama** [reply]
The 0.x branch, which is the branch v0.11.3 is based off, is only compatible with Godot 3.5, the Godot 4 PR is not merged in that branch, only in the master branch. The tag is incorrect, I am not sure how to fix it to be honest.

**Reply by theofficialgman** [reply]
yeah your tag is incorrect. you should fix that either through github web interface or the git cli so that v0.11.3 tag refers to this commit https://github.com/Orama-Interactive/Pixelorama/commit/ac1a438890a58b76bb85f825d9185949b5ce90e8 and not this commit which is entirely incorrect https://github.com/Orama-Interactive/Pixelorama/commit/76e8dfeaaf56bceeb6cff10c2fa980d0b3a88a45

**Reply by theofficialgman** [reply]
without me having to explain it to you, this stackoverflow post goes into enough detail
https://stackoverflow.com/questions/8044583/how-can-i-move-a-tag-on-a-git-branch-to-a-different-commit
basically the process is just, delete the tag locally, create the tag locally on the correct commit and keep the name v0.11.3, and then push the tag with -f which will cause it to overwrite the tag with the corrected tag

**Reply by OverloadedOrama** [reply]
Okay I *think* is should be fixed now. Thanks a lot for the help!

**Reply by theofficialgman** [reply]
yup, looks good!
